### PR TITLE
Fix GA transfer and extrinsic submission by fixing polkadot package v…

### PR DESCRIPTION
**Pull request by KarishmaBothara**
…ersion to use specific package

### CODE REVIEW

**Comment by benxgao** [comment]
LGTM, except fixing linting errors. 

Just wondering if it is just fixed for GA page, but it was showing transaction was finalised in last week, what does it actually fix?

**Comment by KarishmaBothara** [comment]
> LGTM, except fixing linting errors.
> 
> Just wondering if it is just fixed for GA page, but it was showing transaction was finalised in last week, what does it actually fix?

Till last week they(polkadot) were at version 1.4.0 (or something which we are compatible with).. and they keep continously adding changes and as we had ^ , it uses to get the new latest minor version on running yarn.

**Comment by benxgao** [comment]
I see. Then let's use fixed version number for those resolutions from now on.

**Comment by KarishmaBothara** [comment]
> I am not able to take functional review, but have you manually tested the pages which would be affected by these changes?
> 
> If the pages are generally working, and linting is fixed, could get merged.

Yes, have tested the pages from UI.. I am working on fixing the listing and build issue..

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ RUN apt-get install -y nodejs
 
 WORKDIR /apps
 COPY . .
-
+#COPY /Users/karishma/WebstormProjects/cennznet-js-new/cennznet-js/packages/types/cennznet-types-v0.20.7-dev.2.tgz .
```
Comment (holygits) [inline Dockerfile]: Please remove

File: tsconfig.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "**/build/**/*"
   ],
   "compilerOptions": {
-    "baseUrl": ".",
+    "baseUrl": "./",
```
Comment (holygits) [inline tsconfig.json]: is this necessary?

File: package.json [hunk 3]
```diff
@@ -35,6 +36,7 @@
     "start": "cd packages/apps && webpack --config webpack.config.js"
   },
   "devDependencies": {
+    "@cennznet/types": "1.0.0-beta.1",
```
Comment (holygits) [inline package.json]: Why is cennznet/types a devDependency? 🤔

**Comment by KarishmaBothara** [comment]
> LGTM, except fixing linting errors.
> 
> Just wondering if it is just fixed for GA page, but it was showing transaction was finalised in last week, what does it actually fix?

No.. If you try to use the current develop branch, you won't be able to run the application due to dependency issue. To fix that as well as make everything work, we have this branch

**Comment by benxgao** [comment]
I am not sure if how much changes after my last review, but it seems `cennznet-ui` and `cennznet/api.js`  do not have to depend on same version of `polkadot/api.js`. If in that case, could we make a pure PR by just rebasing from `polkadot/apps`? After that, test and fix in a further separate PR if any issues?

**Comment by KarishmaBothara** [comment]
> I am not sure how much changes made after my last review, but it seems that `cennznet-ui` and `cennznet/api.js` do not have to depend on same version of `polkadot/api.js`. If in that case, could we make a pure PR by just rebasing from `polkadot/apps`? After that, test and fix in a further separate PR if any issues?

I checked the develop branch, it seems to work for extrinsic/ga.. I think we can decline this for now... We can plan on how often we need to rebase.

**Pull request closed without merging**